### PR TITLE
handle pkgs with no timestamps

### PR DESCRIPTION
**Pull request by jayfurmanek**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/main/tests/) to validate this change?  

## Description

Some packages in defaults don't have timestamps!
For example:   `cookies-2.2.1-py36_0.tar.bz2`

This checks for a timestamp before doing checks with it.

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: open_ce/conda_utils.py [hunk 1]
```diff
@@ -102,8 +102,9 @@ def conda_package_info(channels, package):
         _, file_name, rest = entry.partition("file name")
         if file_name:
             entry = yaml.safe_load(file_name + rest)
-            # Convert time string into a timestamp
-            entry["timestamp"] = datetime.timestamp(datetime.strptime(entry["timestamp"], '%Y-%m-%d %H:%M:%S %Z'))
+            # Convert time string into a timestamp (if there is a timestamp)
+            if "timestamp" in entry:
+              entry["timestamp"] = datetime.timestamp(datetime.strptime(entry["timestamp"], '%Y-%m-%d %H:%M:%S %Z'))
```
Comment (bnemanich) [inline open_ce/conda_utils.py:107]: Maybe set `entry["timestamp"] = 0` if there is no timestamp. That way you won't have to handle it anywhere else.

Comment (bnemanich) [reply open_ce/conda_utils.py:107]: Also, please add a test to handle this. You can probably just modify one of the existing samples by removing one of the timestamps.

Comment (jayfurmanek) [reply open_ce/conda_utils.py:107]: ah, thats a good idea.

File: tests/validate_config_test.py [hunk 2]
```diff
@@ -41,6 +41,20 @@ def conda_search_result():
 subdir      : linux-ppc64le
 url         : https://repo.anaconda.com/pkgs/main/linux-ppc64le/
 md5         : 4691146ff587f371f83f0e7bab93b63b
+dependencies:
```
Comment (jayfurmanek) [inline tests/validate_config_test.py:44]: Added a no timestamp version here too, though its not really a problem when doing env file validation at the moment.

**Review by bethune-bryant — APPROVED** [review]
LGTM
Looks like it will prefer results that have timestamps, if there's ever one that doesn't and one that does, which is probably good.

**Comment by jayfurmanek** [comment]
yeah - at this point we''re assuming only mistaken or old out of date packages have missing timestamps.

**Pull request merged**